src/shared/components/msktable/MSKTable.tsx

```diff
@@ -397,6 +403,23 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
                 headers={this.store.headers}
                 rows={this.store.rows}
             />
+
+            <If condition={this.store.showingAllRows === false}>
+            <ButtonToolbar style={{marginLeft:0}} className='text-center'>
+                <PaginationControls
```

Comment:
Most of the options for this component is the same as the one on the top. Maybe we should we create a function which returns a `PaginationControls` component. We can pass `style` and `className` as parameters.

Also, would it be better to try to handle styling with the `className` property instead of introducing a new `style` property?